spec/index.bs

```diff
@@ -6997,8 +7047,8 @@ interface GPUQueue {
       GPUImageDataLayout dataLayout,
       GPUExtent3D size);
 
-    undefined copyImageBitmapToTexture(
-        GPUImageCopyImageBitmap source,
+    undefined copyToTexture(
```

Comment:
any reason not to call this `copyPlatformImageToTexture` for consistency?

Reply:
IMO the dictionary name is kludgy but I didn't worry about it because it's not JS-exposed. TBH it's awkward to say "platform image" in this context because... it's all the same platform (web platform). Unfortunately "DOM" doesn't work because ImageBitmap and OffscreenCanvas are not DOM. Not sure what name would be better.

Reply:
(And I think copyToTexture is a good name for the method)

Reply:
The idea here is that we take something from outside of WebGPU and copy the contents into a `GPUTexture`. The `copyToTexture` method doesn't reflect this at all. Would the name `copyExternalImageToTexture` work for this?

Reply:
Discussed in editors meeting. I am OK with this name. It overlaps with the naming of "external textures" but I think that's okay: an "external image" is a canvas/video/etc.; an "external texture" is a WebGPU texture that wraps a canvas/video/etc. It is okay if the actual sets are different (e.g. canvas/video/ImageBitmap vs canvas/video) as those can be thought of as limitations of the entrypoints, not as differences in the definition of "external image". cc @Kangz

Reply:
Although one more thought: "Image" might too strongly imply _just_ HTMLImageElement/ImageBitmap/ImageData and not canvas/video. Will give this a little more thought.